templates/rhpam710-authoring-ha.yaml

```diff
@@ -346,7 +346,7 @@ parameters:
 - displayName: KIE Server Container Memory Request
   description: KIE Server Container memory Request.
   name: KIE_SERVER_MEMORY_REQUEST
-  value: 1.5Gi
+  value: 1500Mi
```

Comment:
on operator we updated it to 1536Mi, can please you apply it here as well?

Reply:
Changed :)